### PR TITLE
feat: export exportToClipboard util from package

### PR DESCRIPTION
**Pull request by ad1992**
https://user-images.githubusercontent.com/11256141/165487436-7f7a1fc3-9746-4c3f-9f38-a1d63ed82df3.mp4

~Do we want to introduce `type`  (svg, png, text) ?~ done

- [x] update docs
- [x] Added type (`png` | `svg` |`text`) and updated the example

### CODE REVIEW

File: src/packages/utils.ts [hunk 1]
```diff
@@ -156,6 +164,33 @@ export const exportToSvg = async ({
   );
 };
 
+export const exportToClipboard = async (
+  opts: ExportOpts & {
+    mimeType?: string;
+    quality?: number;
+    type: "png" | "svg" | "text";
```
Comment (dwelle) [inline src/packages/utils.ts:171]: I'd call it `json` instead of `text`. Text we may reserve for `copyAsText` action?

Comment (ad1992) [reply src/packages/utils.ts:171]: Are you referring to `copy as text` option which shows up in context menu for copying on text elements?

Comment (dwelle) [reply src/packages/utils.ts:171]: yep. But `json` also makes sense in general, since that's how we refer to this type of export everywhere.

Comment (ad1992) [reply src/packages/utils.ts]: done

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
wait

**Comment by ad1992** [comment]
> I fixed not passing `files` in the example app

Sure, thanks 👍🏻

**Pull request merged**